CRM/Core/Payment/Dummy.php

```diff
@@ -146,6 +146,14 @@ public function supportsRefund() {
     return TRUE;
   }
 
+  /**
+   * Supports altering future start dates
+   * @return bool
+   */
+  public function supportsFutureRecurStartDate() {
```

Comment:
This just makes it so that when your using the dummy processor you can expose the pledge date field on contribution front page